build/parser.mjs

```
@@ -446,19 +449,27 @@ class Parser {
     if (item.parent) return;
     // eslint-disable-next-line no-restricted-syntax
-    for (const type of types) {
-      const contains = type.regex ? new RegExp(type.id, 'ig').test(item.uniqueName) : item.uniqueName.includes(type.id);
-      if (contains) {
-        if (type.append) item.type = `${item.type}${type.name}`;
-        else item.type = type.name;
-        // if (item.type !== type.name) console.error(`${item.name} didn't update types`)
-        break;
-      }
+    if (data.wikia.arcanes.find((entry) => entry.name === item.name)) {
+      item.type = data.wikia.arcanes.find((entry) => entry.name === item.name).type
```

ESLint

[error] Insert `;`

(prettier/prettier)